imagehash/__init__.py

```diff
@@ -116,10 +116,10 @@ def average_hash(image, hash_size=8):
 def phash(image, hash_size=32):
 	image = image.convert("L").resize((hash_size, hash_size), Image.ANTIALIAS)
 	pixels = numpy.array(image.getdata(), dtype=numpy.float).reshape((hash_size, hash_size))
-	dct = scipy.fftpack.dct(pixels)
-	dctlowfreq = dct[:8, 1:9]
-	avg = dctlowfreq.mean()
-	diff = dctlowfreq > avg
+	dct = scipy.fftpack.dct(scipy.fftpack.dct(pixels, axis=0), axis=1)
+	dctlowfreq = dct[:8, :8]
```

Comment:
Hi,

I'm a bit confused about this size parameter. None of the solutions seems to be acting consistently with the rest of the algorithms. In *average_hash* and *dhash* functions the **hash_size** parameter defines length of the one side of the square matrix.

```python
hash = imagehash.average_hash(img, hash_size=8)
print(hash.hash.size, hash.hash.shape)
>> (64, (8, 8))

hash = imagehash.dhash(img, hash_size=8)
print(hash.hash.size, hash.hash.shape)
>> (64, (8, 8))
```

Suggestions by both **Queatz** ```python dctlowfreq = dct[:hash_size/4, 1:hash_size/4+1]``` and **wyverntiger** ```python dctlowfreq = dct[:int(hash_size/4), :int(hash_size/4)]``` produce unexpected sizes.
```python
hash = imagehash.phash(img, hash_size=8)
print(hash.hash.size, hash.hash.shape)
>> (4, (2, 2))
```
Would it be possible for the functions to act consistently? Am I missing the point? Thanks.